.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -29,45 +24,44 @@ body:
   - type: textarea
     id: expected-behavior
     attributes:
-      label: Expected Behavior
+      label: Expected behavior
       description: What did you expect to happen?
-      value: |  
-        Expected Behavior:
-        How you expected thing to work.
+      placeholder: |  
+        Expected behavior:
     validations:
       required: false
   - type: input
     id: lakefs_version
     attributes:
-      label: lakeFS Version
+      label: lakeFS version
       description: What version of lakeFS are you running?
   - type: input
     id: deployment
     attributes:
-      label: Deplyoment
-      description: lakeFS deployment type
-      placeholder: ex. AWS ECS, Docker or Kubernetes, helm-chart, local
+      label: How lakeFS is installed
+      description: How and where you installed lakeFS
+      placeholder: ex. AWS EC2, Docker or Kubernetes, locally
     validations:
       required: false
   - type: input
     id: clients
     attributes:
-      label: Affected Clients
-      description: If applicable provide lakeFS client and version that was used
-      placeholder: ex. hadoop-lakefs 0.1.10, python lakefs-client 0.89.0
+      label: Affected clients
+      description: If applicable, provide the lakeFS client and version that was used
+      placeholder: Python lakeFS client 0.89.0
```

Maybe put in a known-_bad_ client version?  That way we know if it wasn't filled out.
```suggestion
      placeholder: Some client 0.0.1-unknown
```

---

I prefer the specific example.